CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Window state is saved on close and restored on start.
 - Files without a defined external file type are now directly opened with the default aplication of the operating system
 - We streamlined the process to rename and move files by removing the confirmation dialogs.
+- We removed the redundant new lines of marking and wrapped the summary in File annotation tab.
```

Comment:
Slightly better: We removed the redundant new lines of marking*s* and wrapped the summary in *the* File annotation tab.

Please also add a reference to the issue #3823 (cf. e.g. some of the changelog entries under "fixed").

Reply:
Fixed.